### DISCUSSION TITLE
Where does "savings rate" fit in to the Firefly III philosophy/vision?

### DISCUSSION THREAD

**Discussion by bradsk88** [firefly-iii/firefly-iii#7073] [General]
As far as I can tell, Firefly III is centralized around the notion that "budgeting is a good financial habit".

Another good habit is to achieve a savings rate as a percentage of your income. But, firefly is missing a handful of things that you'd need to achieve that:
- The ability to record "income" (net or gross)
- The ability to classify expenses/transfers as "savings" (not a tag or a budget, but a native classification)
- The ability to view the report of current savings rate and rate over time.

Has there been any discussion about making this part of Firefly?

**Comment by JC5** [comment]
To answer your main question, it's a nice idea but I'm not really interested in adding it. If you want, you can use tags to tag transactions as income or savings and use the API to add something together.
